README.md

```diff
@@ -23,28 +23,41 @@ Check the [wiki][] for more tips
 Installation
 ------------
 
-1. Using Package Control, install "FileDiffs"
+### Package Control (Easiest)
 
-Or:
+1. Using Package Control, install `FileDiffs`
+
+### Sublime Text 3
 
 1. Open the Sublime Text Packages folder
-    - OS X: ~/Library/Application Support/Sublime Text 3/Packages/
-    - Windows: %APPDATA%/Sublime Text 3/Packages/
-    - Linux: ~/.Sublime Text 3/Packages/ or ~/.config/sublime-text-3/Packages
+    - OS X: `~/Library/Application Support/Sublime Text 3/Packages/`
+    - Windows: `%APPDATA%/Sublime Text 3/Packages/`
+    - Linux: `~/.Sublime Text 3/Packages/` or `~/.config/sublime-text-3/Packages`
+
+2. Clone this repo
+
+    ```
+    # Over SSH
+    git clone git@github.com:colinta/SublimeFileDiffs
+
+    # Over HTTPS
+    git clone https://github.com/colinta/SublimeFileDiffs.git
+    ```
 
-2. clone this repo
 3. Install keymaps for the commands (see Example.sublime-keymap for my preferred keys)
```

Comment:
I like the emphasis on installing via package control.  This step - adding keymaps - should be included in that section, because it needs to be added no matter how you install the package.